src/edu/csus/ecs/pc2/ui/ReportPane.java

```diff
@@ -486,9 +486,9 @@ protected void createHeader(PrintWriter printWriter, IReport report) {
             resumeTime = getContest().getContestTime().getResumeTime();
         }
         if (resumeTime == null) {
-            printWriter.print("  Contest date/time: never started");
+            printWriter.print("  Contest date/time: never started on site "+getContest().getSiteNumber());
         } else {
-            printWriter.print("  Contest date/time: " + resumeTime.getTime());
+            printWriter.print("  Contest date/time: " + resumeTime.getTime()+" on site "+getContest().getSiteNumber());
```

Comment:
The same problem appears to exist here (`getContest().getSiteNumber()` could throw an NPE).